compiler/test/E2E/Host/AliasLikeGPU/1_hlo_opt.mlir

```diff
@@ -0,0 +1,14 @@
+// RUN: byteir-opt %s -hlo-graph-opt -hlo-fusion-opt="outline-single-elemwise-op" | FileCheck %s
```

Comment:
Should move `AliasLikeGPU` to `E2E/CUDA`?

Reply:
> Should move `AliasLikeGPU` to `E2E/CUDA`?

I found the test cases under `E2E/CUDA` are complete nn models, that's why I moved `alias` case to host side.

Reply:
I think it's better to put it under `E2E/CUDA`. Also should add it to `gen_testcases_and_check_diff.sh`.